crontinuous_test.go

```diff
@@ -332,6 +332,137 @@ func TestExecutesEntries(t *testing.T) {
 	}
 }
 
+func TestCrontinuous_isProgramSuffixIncluded(t *testing.T) {
+	tests := []struct {
+		name            string
+		programID       string
+		programSuffixes string
+		want            bool
+	}{
+		{
+			name:            "Empty suffixes",
+			programID:       "program@web-scanning",
+			programSuffixes: "",
+			want:            false,
+		},
+		{
+			name:            "One prefix matching",
+			programID:       "program@redcon-scan",
+			programSuffixes: "@redcon-scan",
+			want:            true,
+		},
+		{
+			name:            "One prefix not matching",
+			programID:       "program@redcon-scan",
+			programSuffixes: "@periodic-full-scan",
+			want:            false,
+		},
+		{
+			name:            "Two prefixes, one matching",
+			programID:       "program@redcon-scan",
+			programSuffixes: "@redcon-scan,@periodic-full-scan",
+			want:            true,
+		},
+		{
+			name:            "Two prefixes, one matching",
+			programID:       "program@redcon-scan",
+			programSuffixes: "@redcon-scan,@periodic-full-scan",
+			want:            true,
```

Comment:
I think this test is duplicated it is the same than the test above, isn't it?

Reply:
Absolutely right. Removed here [a732854](https://github.com/adevinta/vulcan-crontinuous/pull/19/commits/a732854aefc9370ebca0ced91b2b5db52a3658e1)